python/src/gen_gherkin/generate_gherkin_feature_files_impl.py

```diff
@@ -58,14 +58,16 @@ def gen_gherkin_behaviors(
 
     yaml_list = []
     for model in results:
+        behavior_list = []
```

Comment:
Sure this is needed? See you add it here but don't see it referenced in your other changes.